components/gpio_control/README.md

```diff
@@ -161,6 +163,8 @@ functionCallTwoButtons: functionCallVol0
 
 In this example, the volume will be in-/decreased step-wise using intervals of 0.3 seconds while the respective button is held. If both buttons are pushed simultaneously, the player is muted (volume 0). In this example, Pin1 is used for increasing the volume, while Pin2 decreases it.
 
+Function arguments can also be passed to this control with the settings **functionCall1Args**, **functionCall2Args**, **functionCallTwoButtonsArgs**
+
 Furthermore, the following settings can be used as described for the [regular buttons](#doc_button): **pull_up_down**, **edge**, **bouncetime**, **antibouncehack**
 
 ### RotaryEncoder<a name="doc_rotary"></a>
```

Comment:
Do we still need html anchors? I think markdown can do this now with appending `#RotaryEncoder` in a link.

Reply:
I wondered what they are for. So its legacy? then we can remove them :)

Reply:
I think removing should be fine